tests/classy_jest/_support/mockApiTickets.ts

```diff
@@ -0,0 +1,62 @@
+import { GetTicketApiResponse } from '@tec/tickets/classy/types/Api';
+
+/**
+ * Mock API tickets data for testing.
+ *
+ * @return {GetTicketApiResponse[]} The mock API tickets array.
+ */
+export function makeMockApiTickets(): GetTicketApiResponse[] {
```

Comment:
This look like a repetition of the data from the `makeMockApiResponse` function from the `mockApiResponse.ts` file.
I would update the function in `mockApiResponse.ts` to offer customization parameters and use it to build this array.